### DISCUSSION TITLE
v2.9.11新创建的 <option> 元素显示在顶部 无效

### DISCUSSION THREAD

**Discussion by hjingsuper** [layui/layui#1984] [讨论]
> v2.9.11开启 lay-creatable 时，新创建的 <option> 元素显示在顶部

这个调整好像没有作用，显示在顶部需要什么属性嘛
![2024-06-06-08-56-13](https://github.com/layui/layui/assets/36070786/23c86ed9-9bc1-4f61-919b-a7ba5ccfd869)

**Comment by Sight-wcg** [comment]
新创建的 option 元素，创建过程中显示在顶部，方便选取，创建完成后，仍然添加到 option 列表最后面。

**Reply by hjingsuper** [reply]
哦哦，有点误解了。谢谢解答
